autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleScalingActivityV1Controller.java

```diff
@@ -38,10 +38,18 @@ public class DistroXAutoScaleScalingActivityV1Controller implements DistroXAutoS
 
     @Override
     @CheckPermissionByResourceCrn(action = AuthorizationResourceAction.DESCRIBE_DATAHUB)
-    public DistroXAutoscaleScalingActivityResponse getScalingActivityUsingOperationId(@ResourceCrn @TenantAwareParam String clusterCrn, String operationId) {
+    public DistroXAutoscaleScalingActivityResponse getScalingActivityUsingOperationIdAndClusterCrn(@ResourceCrn @TenantAwareParam String clusterCrn,
+            String operationId) {
         return distroXAutoscaleScalingActivityResponseConverter.convert(scalingActivityService.findByOperationIdAndClusterCrn(operationId, clusterCrn));
     }
 
+    @Override
+    @CheckPermissionByResourceName(action = AuthorizationResourceAction.DESCRIBE_DATAHUB)
+    public DistroXAutoscaleScalingActivityResponse getScalingActivityUsingOperationIdAndClusterName(@ResourceName @TenantAwareParam String clusterName,
```

Comment:
TenantAwareParam work only on crn parameters, pls check https://github.com/hortonworks/cloudbreak/tree/master/authorization-common#how-to-make-an-api-internally-callable for further information

Reply:
so you can simply remove the annotation if you do not need to call this internally, if internal call needed, then you should add an extra account id parameter with `@AccountId` annotation, as described in the authz readme linked above

Reply:
I have removed TenantAwareParam and updated the PR.